modules/UI/authentication/AuthHandler.js

```diff
@@ -80,8 +81,11 @@ function initJWTTokenListener(room) {
             return;
         }
         if (event.data && event.data.jwtToken) {
-            config.token = event.data.jwtToken;
-            logger.info("Received JWT token:", config.token);
+            const token = event.data.jwtToken;
+
+            APP.store.dispatch(updateConfig({ token }));
```

Comment:
Sure, I think we can even get rid of it, because we're storing the same data in token data already.